Prepare

```diff
@@ -5,5 +5,5 @@ set -e
 
 sh ./autogen.sh
 
-CFLAGS="-O0 -g -Wall" ./configure --with-pic --enable-maintainer-mode --enable-dependency-tracking --enable-shared=yes
+CFLAGS="-O0 -g -Wall" ./configure --with-pic --enable-dependency-tracking --enable-shared=yes
```

Comment:
Here's a funny fact:

Makefile.am.common comes with an option that turns dependency-tracking off for every Makefiles.am that includes it. *ALL* Makefiles (that do anything meaningful) include Makefile.am.common. So dependency-tracking is off, right?
Well, yes, but only after the fact.

Autoreconf first goes through all ins and outs of configure.ac, can't find anything that suggests that dependency-tracking is off, and spit out a configure script that accepts the --enable-dependency-tracking option.
If you choose to enable dependency-tracking this way, configure even goes ahead and happily runs tests to find out what it takes to make it happen. When it reaches the end, and is populating Makefiles, the AUTOMAKE_OPTIONS rule in Makefile.am.common kicks in and silently disables dependency-tracking.

During dependency-tracking, a hidden subdirectory called .deps is created that holds dependency output files with the same name as all processed object files and they all have a suffix of .Plo.
This "Prepare" script, added in 2009, ask for dependency-tracking but it never  happened. You know why I know this? Because there's no trace of either in .gitignore.

Figuring this out has cost an entire afternoon and I don't even need it. What a waste...